packages/paste-website/src/pages/index.tsx

```diff
@@ -39,7 +39,7 @@ const IndexPage: React.FC<{}> = (): React.ReactElement => {
         <meta name="description" content={SiteMetaDefaults.DESCRIPTION} />
       </Helmet>
       <LandingContainer maxWidth="size60" marginTop="space200" marginBottom="space200">
-        <Heading as="h1" headingStyle="headingStyle10">
+        <Heading as="h2" headingStyle="headingStyle10">
```

Comment:
What's the reasoning behind bumping the heading Heirarchy down one? There's now no `h1` on the landing page.

Reply:
I noticed that component pages were using `h1` for the component header so this one would clash with that.  Thinking on it, I figured the component page h1 is more meaningful as it changes from page to page.  I don't think setting h1 on `Paste` is very meaningful semantically, and the SEO should be fine as `Paste` appears in the URL and pretty much everywhere a bunch already.

Reply:
So, we need one per page. Even if we change the main title from Paste to something else. Even if we don't think it's a good title for the page. You could even hide some text inside of it to make it more meaningful but keep the visuals the same, but we do need one. Lighthouse complains and it'll be the first thing someone picks up on from an Accessibility audit.

Reply:
You're totally right 😅 I mistook this page for the `Paste` in the sidebar.  I'll revert this.

Reply:
ooooooooooooohhhhhh 🤣  Yeah, that being an `h2` also bugs me, but not in the scope of this work